src/main/resources/com/xebialabs/xlrelease/ci/Credential/config.jelly

```diff
@@ -10,6 +10,11 @@
     <f:entry title="${%Password}" field="password">
         <f:password/>
     </f:entry>
+    <f:optionalBlock inline="true" field="useGlobalCredential" title="Use stored credentials" checked="${instance.showGolbalCredentials()}">
+        <f:entry title="${%Credentials}" field="credentialsId">
+            <c:select/>
+        </f:entry>
+    </f:optionalBlock>
```

Comment:
@angel-git there is no such property available in jelly to do so. 

Nadia gave us a suggestion to completely hide the username and password if Use stored credentials checkbox clicked. we are working in that direction  

If you have implemented something like this before please help.